server/controllers/stock/inventory.js

```diff
@@ -167,6 +170,37 @@ function getInventoryItemsById(req, res, next) {
   .done();
 }
 
+/**
+* GET /inventory/metadata/receipt
+* Returns a pdf file for inventory metadata
+*
+* @function getInventoryItemReport
+*/
+function getInventoryItemReport(req, res, next) {
+  'use strict';
+
+  let request = {
+    query : req.query,
+    enterprise : req.session.enterprise,
+    project : req.session.project
+  };
+
+  core.getItemsMetadata()
+  .then((rows) => {
+    return listReceipt.build(rows, request);
+  })
+  .then((result) => {
+    let renderer = {
```

Comment:
Since `renderer` is not changing, it might be better to put it as a constant outside of the function call.  Like this:
```js
// response headers for reports
const headers = {
  'pdf' : '"Content-Type" : "application/pdf"
  // ... etc .
};

// the report function
function getInventoryItemReport() { /* ... */ }
```

I noticed that the `renderer` is actually just an object of headers, and it can be called `headers` instead.

Finally, it is worth noting that this can also be done without using strings:
```js
const headers = {
  pdf : { 'Content-Type' : 'application/pdf' },
  html : { 'Content-Type'  : 'application/html' },
};
```
also works.  [Link to the documentation](http://expressjs.com/en/4x/api.html#res.set).